tuf/src/repository/track_repo.rs

```diff
@@ -117,8 +117,7 @@ where
             let mut buf = Vec::new();
             metadata.read_to_end(&mut buf).await?;
 
-            let () = self
```

Comment:
i actually like these unit bindings, makes it very obvious to the reader nothing important is being ignored (i know `must_use` is for that, but sometimes there are exceptions

not that you shouldn't get rid of it here, just be prepared for push back in //src/sys/pkg :)